src/Core/src/Platform/Android/JavaObjectExtensions.cs

```diff
@@ -30,5 +33,18 @@ public static bool IsAlive([NotNullWhen(true)] this global::Android.Runtime.IJav
 
 			return !obj.IsDisposed();
 		}
+
+		public static TResult? TryJavaCast<[DynamicallyAccessedMembers (Constructors)] TResult>(this IJavaObject? instance)
+			where TResult : class, IJavaObject
+		{
+			try
+			{
+				return instance.JavaCast<TResult>();
+			}
+			catch
+			{
+				return null;
+			}
```

Comment:
I'm not a fan of an empty try-catch with no logging.

Does a `[DynamicDependency]` like @simonrozsival suggested also solve it? That would have no performance cost, and I think things would still be trimmed away if you had 0 animated gifs in your app.

Reply:
I logged an issue with a feature request or something: https://github.com/dotnet/android/issues/9038